### PR TITLE
limit working now with binomial #10801 

### PR DESCRIPTION
**Pull request by wrat**
As we know that binomial funtion can be reduced into to their factorial form
 for calculating the limit of funtion we are using gruntz algorithm. gruntz algorithm is not works with  binomial limits so we have to convert its factorial form.
 for example:
 while calculating `limit(16**k/(k*binomial(2*k,k)**2), k, oo)`
 we are getting NotImplemented Exception but after some changes we are now able to get the result as PI.
"

### CODE REVIEW

File: sympy/series/limits.py [hunk 1]
```diff
@@ -135,7 +136,12 @@ def doit(self, **hints):
 
         if not e.has(z):
             return e
-
+        #limit is not working with binomial,gruntz raise an exception
```
Comment (smichr) [inline sympy/series/limits.py]: space after `#` and space around `=` below

Comment (wrat) [inline sympy/series/limits.py]: So it is ready  now to get merged ?

**Comment by smichr** [comment]
Please edit your commit message:

:eyeglasses: see requirements [here](https://github.com/sympy/sympy/wiki/Development-workflow#writing-commit-messages)

**Comment by smichr** [comment]
Other than the comments I made, this seems like a good idea. Thanks for identifying this improvement.

File: sympy/series/limits.py (hunk 1 above)
Comment (wrat) [inline sympy/series/limits.py]: what else should I add in my commit message??

Comment (smichr) [inline sympy/series/limits.py]: > ready now to get merged

No yet. Please add spaces after the `#` and format your commit message with a blank line after the title. Please see the reference/link that I referred to if you have questions.

**Comment by wrat** [comment]
@asmeurer ping!!

File: sympy/series/limits.py [hunk 2]
```diff
@@ -136,12 +136,13 @@ def doit(self, **hints):
 
         if not e.has(z):
             return e
-        #limit is not working with binomial,gruntz raise an exception
+
```
Comment (smichr) [inline sympy/series/limits.py]: I didn't mean put line breaks after the `#` I meant to change `#foo` to `# foo` -- space after the `#`

a space after the `binomial,`, too

Comment (wrat) [inline sympy/series/limits.py]: ping @smichr

**Comment by wrat** [comment]
ping @asmeurer

**Comment by leosartaj** [comment]
Closing in favor of #10883.

**Pull request closed without merging**